Source/WebInspectorUI/UserInterface/Views/SourcesNavigationSidebarPanel.js

```diff
@@ -1877,7 +1877,7 @@ WI.SourcesNavigationSidebarPanel = class SourcesNavigationSidebarPanel extends W
             if (pauseData.eventListenerId)
                 eventBreakpoint = WI.domManager.breakpointForEventListenerId(pauseData.eventListenerId);
             if (!eventBreakpoint)
-                eventBreakpoint = WI.domDebuggerManager.listenerBreakpointForEventName(pauseData.eventName);
+                eventBreakpoint = WI.domDebuggerManager.listenerBreakpointsForEventName(pauseData.eventName)[0];
```

Comment:
Are we positive there will always be a breakpoint in the returned array?

Reply:
we know 100% that there will always be an array, but yes we don't know for sure that there will be at least one item (though there should be in order for us to reach this point, as we need to have an event breakpoint somewhere in order to pause at an event).  this is why we check `if (!eventBreakpoint)` below

Reply:
I guess my actual question is are we sure enough there will be a breakpoint to assert when there isn't one. The answer seems to be yes.